README.md

```diff
@@ -39,70 +41,48 @@ postprocessing.  Its features include:
+
+Trixi and related postprocessing tools are registered Julia packages. Hence, you
+can install Trixi, [Trixi2Vtk](https://github.com/trixi-framework/Trixi2Vtk.jl),
+and [Trixi2Img](https://github.com/trixi-framework/Trixi2Img.jl) by executing
+the following commands in the Julia REPL:
 ```julia
 julia> import Pkg
 
-julia> Pkg.add("Trixi")
-```
-If you do this and want to modify Trixi, you can run
-```julia
-julia> Pkg.dev("Trixi") # get a clone of the git repository, usually in ~/.julia/dev/Trixi
+julia> Pkg.add("Trixi"); Pkg.add("Trixi2Vtk"); Pkg.add("Trixi2Img")
 ```
+Further details can be found in the [documentation](#documentation), including
+instructions on how to install Trixi such that you can modify or extend it with
+your own implementations.
 
 
 ## Usage
-Enter the root directory `Trixi.jl/` and execute
-```bash
-julia --project=@.
-```
-This will start an interactive Julia session (REPL) using the project setup
-of Trixi.jl. If you have installed Trixi.jl in your default project environment,
-you can just start Julia as usual
-```bash
-julia
-```
-In the Julia REPL, you need to load the package Trixi
+In the Julia REPL, first load the package Trixi
 ```julia
 julia> using Trixi
 ```
-To run a simulation, execute
+Then, to start a simulation, execute
 ```julia
-Trixi.run("examples/parameters.toml")
-```
-You can also pass a different parameters file or edit `examples/parameters.toml` to
-modify the simulation setup.
-
-Sometimes it can be helpful to run Trixi non-interactively in batch mode, e.g.,
-when starting a simulation from another script. This is possible by directly passing
-the code that shall be executed to Julia
-```bash
-julia -e 'using Trixi; Trixi.run("examples/parameters.toml")'
+Julia> Trixi.run(default_example())
 ```
+The `run(...)` method expects a single string argument with the path to a
+Trixi parameter file. To quickly see Trixi in action, `default_example()`
+returns the path to an example parameter file with a short, two-dimensional
+problem setup. A list of all example parameter files packaged with Trixi can be
+obtained by running `get_examples()`. Alternatively, you can also browse the
+[`examples/`](examples/) subdirectory. If you want to
+modify one of the parameter files to set up your own simulation, download it to
+your machine, edit the configuration, and pass the file path to `run(...)`.
 
 
 ## Documentation
 Additional documentation is available that contains more information on how to
-use Trixi interactively, how to visualize output files etc. It also includes a
-section on our preferred development workflow and some tips for using Git. The
-latest documentation can be accessed either
+modify/extend Trixi's implementation, how to visualize output files etc. It
+also includes a section on our preferred development workflow and some tips for
+using Git. The latest documentation can be accessed either
 [online](https://trixi-framework.github.io/Trixi.jl/dev) or under [`docs/src`](docs/src).
```

Comment:
Do we really want to point to the dev docs here instead of the stable docs? If a user has installed Trixi as described, they need the stable docs.

Reply:
Fixed in f4e4e1c.